resources/popup.html

```diff
@@ -39,7 +40,7 @@
                 </a>
             </div>
             <div class="col s6">
-                <a class="btn waves-effect lime waves-light tooltipped" href="../analytics/index.html" target="_blank"
+                <a id="analytics-link" class="btn waves-effect lime waves-light tooltipped" target="_blank"
```

Comment:
Yes it was already there. But not in `options.html`, because I don't think we need a new tab from the options page. From a popup we need to do this, because the analytics page won't fit in the popup. Therefore the target was already set here, but not in `options.html` or `main.js`